Scripts/extras/pull_and_build_from_git.py

```diff
@@ -325,11 +338,15 @@ def __init__(self, package_info, platform_config, base_folder, build_folder, pac
         self.build_install_folder = self.package_install_root / package_info.package_name
         self.clean_build = clean_build
         self.cmake_find_template = cmake_find_template
+        self.cmake_find_source = cmake_find_source
         self.build_configs = platform_config.get('build_configs', ['Debug', 'Release'])
         self.prebuilt_source = prebuilt_source
         self.prebuilt_args = prebuilt_args
         self.skip_git = skip_git
 
+        if self.cmake_find_template is not None and self.cmake_find_source is not None:
+            raise BuildError("Bad build config file. 'cmake_find_template' and 'cmake_find_template' cannot both be set in the configuration.")            
+
```

Comment:
This check is done in 2 places, PackageInfo::__init__ and BuildInfo::__init__, do we need both?

Additionally PackageInfo checks for both None, do we also need that check here?

Reply:
Yeah, I think PackageInfo::init is all we need. I'll remove the other checks